visualization.py

```diff
@@ -0,0 +1,134 @@
+import cv2
+import numpy as np
+import plotly.express as px
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+from scipy.signal import savgol_filter
+import os
+import data_filtering
+import mediapipe as mp
+
+mp_drawing = mp.solutions.drawing_utils
+mp_drawing_styles = mp.solutions.drawing_styles
+mp_pose = mp.solutions.pose
+
+# For static images:
+BG_COLOR = (192, 192, 192)  # gray
+
+
+def create_annotation(image, results, skeleton):
+    annotated_image = image.copy()
+    bg_image = np.zeros(image.shape, dtype=np.uint8)
+    bg_image[:] = BG_COLOR
+    mp_drawing.draw_landmarks(
+        annotated_image,
+        results.pose_landmarks,
+        mp_pose.POSE_CONNECTIONS,
+        landmark_drawing_spec=mp_drawing_styles.get_default_pose_landmarks_style())
+
+    visualize_landmarks_coordinates(skeleton)
+    annotated_image = concat_visulisations(annotated_image)
+    return annotated_image
+
+
+def concat_visulisations(image):
+    graph_img_x = cv2.resize(cv2.imread("temp_x.png"), (image.shape[1], image.shape[0]))
+    graph_img_y = cv2.resize(cv2.imread("temp_y.png"), (image.shape[1], image.shape[0]))
+    graph_img_z = cv2.resize(cv2.imread("temp_z.png"), (image.shape[1], image.shape[0]))
```

Comment:
Where do these names come from?

Reply:
file names

Reply:
I need them for create visualization
They were created by this pipeline, and removed when pipeline finished

Reply:
It would be nice to store them as constants if they are used elsewhere